clients/nacos_client/nacos_client.go

```diff
@@ -93,6 +93,9 @@ func (client *NacosClient) SetServerConfig(configs []constant.ServerConfig) (err
 		if len(configs[i].ContextPath) <= 0 {
 			configs[i].ContextPath = constant.DEFAULT_CONTEXT_PATH
 		}
+		if configs[i].Scheme == "" {
```

Comment:
use len(configs[i].Scheme)<=0 ，keep the style consistent

Reply:
okay, fine